### PR TITLE
added crazydevlegend

### PR DESCRIPTION
**Pull request by crazydevlegend**
Adding my profile...

## Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1905"></a>

### CODE REVIEW

**Comment by crazydevlegend** [comment]
I just wanna know the meaning of `invalid` tag `github-actions` bot added to this PR.
@EmmaDawsonDev Can you please review my PR and check if it's all right to merge?

**Comment by crazydevlegend** [comment]
Awesome, now what shall i do to make it able to merge, @EmmaDawsonDev ?
or it doesn't have to be merged???

**Comment by EmmaDawsonDev** [comment]
Just wait for a second maintainer review. If they have any comments you should get a message about changes, otherwise they'll approve and merge :)

**Comment by crazydevlegend** [comment]
@Panquesito7 can you please review this PR?
It's nothing, but only to add my profile.
and please let me know if it's able to be merged...

File: public/data/crazydevlegend.json
Comment (Panquesito7) [inline public/data/crazydevlegend.json:4]: I recommend capitalizing the `b` here. What do you think?
```suggestion
  "bio": "Full-stack | Blockchain expert | Open Source Enthusiast!",
```

Comment (crazydevlegend) [reply public/data/crazydevlegend.json]: Yes, agreed, @Panquesito7 
Just fixed it and now ready to merge...

**Review by Panquesito7 — APPROVED** [review]
Great! Thanks. 🎉

**Pull request merged**
